why-promptlayer/shared-workspaces.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: "Shared Workspaces"
+icon: "handshake-simple"
+---
+
+# Overview
+
+The Shared Workspaces feature in PromptLayer is a collaborative tool designed to enhance teamwork and efficiency. It allows users to organize and share various data such as requests, prompt templates, and evaluations, in a central workspace.
+
+## Key Features
+
+- **Data Organization**: Centralize all your team's data in one shared workspace.
+- **Collaboration**: Work together with your team, share resources, and streamline your workflow.
+
+## Getting Started
+
+This feature is available under the “Startup” or “Team” plan. Upgrade your plan by navigating to “Billing” in the settings cog at the top right corner of your PromptLayer account. Once upgraded, you can create a new workspace by clicking the 'Create Workspace' button from the workspace dropdown.
+
+![Create Workspaces Button](/images/workspaces-create-button.png)
+
+Choose a unique name for your workspace to enhance its identity and ease of recognition among team members. A meaningful and memorable name can be a great way to represent the workspace's purpose and facilitate easier collaboration.
```

Comment:
"A meaningful and memorable name can be a great way to represent the workspace's purpose and facilitate easier collaboration."

GPT lol? useless sentence maybe remove